sdk/nodejs/src/eventQueue.ts

```diff
@@ -219,6 +220,12 @@ export class EventQueue {
                     this.logger.debug(
                         `DevCycle Error Flushing Events response message: ${ex.message}`,
                     )
+                    if (ex instanceof ResponseError && ex.status === 401) {
+                        this.logger.debug(
+                            `SDK key is invalid, stopping retrying`,
+                        )
+                        return
```

Comment:
I don't think this would do anything except put the bucketing events queue in a limbo state where the status of the current payload is never updated from pending, while new events continue to accumulate and the flush interval continues to run/

Reply:
there also isn't any "retrying" here, this is just the flush loop that runs on a fixed interval. So the bigger mystery is why it was tight-looping request retries yesterday